components/homme/src/theta-l_kokkos/cxx/CaarFunctorImpl.hpp

```diff
@@ -105,6 +106,8 @@ struct CaarFunctorImpl {
   Kokkos::RangePolicy<ExecSpace, TagPostExchange> m_policy_post;
   Kokkos::TeamPolicy<ExecSpace, TagDp3dLimiter>   m_policy_dp3d_lim;
 
+  mutable TeamUtils<ExecSpace> m_tu;
```

Comment:
Why mutable?

Reply:
Because the functor operator() are const and getting a team-idx is non-const (has to change TeamUtils internals top keep track of state).

Reply:
I don't see where you would violate const. All methods in TeamUtils are const.

Reply:
Good catch. It was just that the KernelVariables member was non-const and needed to be changed to const.